### ISSUE TITLE
is it possible to build neural network with linfa?

### ISSUE THREAD

**Issue by dbsxdbsx** [rust-ml/linfa#211]
I want to build and train a neural network  through gradient descent but with very[ flexible structure](https://en.wikipedia.org/wiki/Neuroevolution_of_augmenting_topologies), Is it  possible to be with `Linfa`?

**Comment by YuhanLiin** [comment]
We only support classical ML methods, such as linear regression and logistic regression, not deep learning methods. This means neural nets are out of scope for this project.
